nototools/generate_website_2_data.py

```diff
@@ -246,8 +246,7 @@ def get_family_id_to_regions(family_id_to_lang_scr_to_sample_key):
 
   family_id_to_regions = collections.defaultdict(set)
   warnings = set()
-  for (family_id, lang_scr_to_sample_key in
-       family_id_to_lang_scr_to_sample_key.iteritems()):
+  for family_id, lang_scr_to_sample_key in family_id_to_lang_scr_to_sample_key.iteritems():
```

Comment:
That is frighteningly obvious, isn't it... Not my day.

On Tue, Sep 29, 2015 at 2:19 PM, Roozbeh Pournader <notifications@github.com
> wrote:

> In nototools/generate_website_2_data.py
> <https://github.com/googlei18n/nototools/pull/92#discussion_r40731287>:
>
> > @@ -246,8 +246,7 @@ def get_family_id_to_regions(family_id_to_lang_scr_to_sample_key):
> >
> >    family_id_to_regions = collections.defaultdict(set)
> >    warnings = set()
> > -  for (family_id, lang_scr_to_sample_key in
> > -       family_id_to_lang_scr_to_sample_key.iteritems()):
> > +  for family_id, lang_scr_to_sample_key in family_id_to_lang_scr_to_sample_key.iteritems():
>
> Try unpacking the tuple inside the loop?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlei18n/nototools/pull/92/files#r40731287>.
>